contracts/schemes/GlobalConstraintAddOrRemove.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity 0.5.17;
+
+import "../controller/Controller.sol";
+
+/**
+ * @title A scheme for adding or removing a global constraint
+ * The scheme will unregister itself after register the globalConstraint
+ * This scheme should be register to the dao with permission 0x00000004
+ */
+
+contract GlobalConstraintAddOrRemove {
```

Comment:
What's this contract for? We already have `GlobalConstraintRegistrar` which does this much better...